src/components/Header/Header.tsx

```diff
@@ -39,7 +39,7 @@ export const Header = chakra(() => {
     isWidthLarge,
   });
 
-  const [loginOpen, setLoginOpen] = useState<boolean>(false);
+  const [isLoginOpen, setIsLoginOpen] = useState<boolean>(false);
```

Comment:
Is it a requirement that an open login should persist between desktop and mobile?  The current NYPL header doesn't do this.

Reply:
Right, it's not. Some of this has been updated in the latest PR so I'll keep iterating on a later PR.